### DISCUSSION TITLE
sentence-transformer库 | 句子语义向量化

### DISCUSSION THREAD

**Discussion by giscus** [hiDaDeng/hidadeng.github.io#46] [交流]
# sentence-transformer库 | 句子语义向量化

使用sentence-transformer库BERT技术，将句子语义向量化

https://textdata.cn/blog/sentence-transformer-tutorial/

**Comment by hoMunCuLusSSSS** [comment]
使用SentenceTransformer的时候会报错，ValueError: Asking to pad but the tokenizer does not have a padding token. Please select a token to use as `pad_token` `(tokenizer.pad_token = tokenizer.eos_token e.g.)` or add a new pad token via `tokenizer.add_special_tokens({'pad_token': '[PAD]'})`.这个token在哪里设置啊？
